Libraries/LibWeb/ContentSecurityPolicy/Directives/Names.h

```diff
@@ -39,6 +39,4 @@ namespace Web::ContentSecurityPolicy::Directives::Names {
 ENUMERATE_DIRECTIVE_NAMES
 #undef __ENUMERATE_DIRECTIVE_NAME
 
-void initialize_strings();
```

Comment:
does names.cpp already follow the flystring pattern used here for keyword sources?

Reply:
Oh I see it was removed b/c of Shannon's comment, carry on then.